docs/api-process/check-out.md

```diff
@@ -1,6 +1,349 @@
 ---
 id: check-out
-title: How to Create Checkout?
+title: How to Create Checkout
 ---
 
-This topic describes the process of check out.
\ No newline at end of file
+## Introduction
+
+Below is a description of a checkout process. We assume that at this stage you have already completed the steps included in the [Getting Started](api/intro.md) section of this chapter and you are familiar with the basic setup of Saleor GraphQL API.
+
+The below process describes the key milestones in the checkout process flow in Saleor. There are also additional steps that may occur along the way, however, the purpose of this instruction is to deliver a base reference for the user to work with.
+
+The code snippets included in this section may be run in [Playground](api/playground.md) or your preferred HTTP client.
+
+## Step #1 - `checkoutCreate`
+
+> **Note**
+>
+> A `Checkout` object can be created for logged in users and for anonymous (guest) users.
+>
+> - If you use the `checkoutCreate` mutation including the authentication token, this checkout is assigned to the user who is authenticated by this token. For more information on how to authenticate with our API, see the [Authentication](api/authenticate.md) topic.
+>
+> - If no authentication token is provided, the checkout is created for an anonymous user, and an email address is used to identify such `Checkout` object linking it with the anonymous user. In this case, an email is required to create checkout.
+
+To create a `Checkout` object, use the `checkoutCreate` mutation.
+
+This mutation requires the following input:
+
+- `email` - user's email address
+
+- `shippingAddress` - user's shipping address
+
+- `billingAddress` - user's billing address
+
+- `lines` - list of checkout lines - each checkout line represents the _variant id_ (the specific product) and the quantity of it.
```

Comment:
It would be nice to use one format for all items in the list. I suggest getting rid of the ending dot and using a semicolon to separate sentences.

```suggestion
- `lines` - list of checkout lines - each checkout line represents the _variant id_ (the specific product) and the quantity of it
```